### ISSUE TITLE
ID inside annotations

### ISSUE THREAD

**Issue by Yash-Varshney** [spdx/spdx-spec#480]
This issue's inspiration is - [here](https://github.com/Yash-Varshney/tools-python/issues/9) ( 7th point)

In the previous version of SPDX (which presently python tool supports), there is `ID` inside `annotations` whereas in v2.2 `ID` is missing. 

* Annotations in previous versions
``` 
"annotations": [
            {
                "comment": "This is just an example. Some of the non-standard licenses look like they are actually BSD 3 clause licenses", 
                "annotationType": "REVIEW", 
                "id": "SPDXRef-45", 
                "annotationDate": "2012-06-13T00:00:00Z", 
                "annotator": "Person: Jim Reviewer"
            }
        ], 
```
* Annotations in latest version 
```
"annotations" : [ {
    "annotationDate" : "2010-01-29T18:30:22Z",
    "annotationType" : "OTHER",
    "annotator" : "Person: Jane Doe ()",
    "comment" : "Document level annotation"
  }, 
```
It might cause problem in providing stability and support to previous versions' parsing.

**Comment by kestewart** [comment]
@goneall - I think we need to transfer this one to the python tools to check?

**Comment by goneall** [comment]
@kestewart In thinking about this, it will change back to an ID in spec version 3.0 since annotations are full elements.

I'm going to move this to the 3.0 milestone.

**Comment by goneall** [comment]
This is resolved in 3.0 - closing
